packages/wrangler/e2e/vitest.config.mts

```diff
@@ -3,17 +3,17 @@ import { defineConfig } from "vitest/config";
 
 export default defineConfig({
 	test: {
-		testTimeout: 120_000,
+		testTimeout: 90_000,
 		poolOptions: {
 			threads: {
 				singleThread: true,
 			},
 		},
-		retry: 2,
 		include: ["e2e/**/*.test.ts"],
 		// eslint-disable-next-line turbo/no-undeclared-env-vars
 		outputFile: process.env.TEST_REPORT_PATH ?? ".e2e-test-report/index.html",
 		globalSetup: path.resolve(__dirname, "./validate-environment.ts"),
 		reporters: ["verbose", "html"],
+		bail: 1,
```

Comment:
From the docs:

> Stop test execution when given number of tests have failed 

So this will stop when the first test fails. The default of zero means to never bail.